.github/pull_request_template.md

```diff
@@ -1,39 +1,11 @@
-Notes:
-- Pull requests will not be accepted until the submitter has agreed to the [contributer agreement](https://github.com/ARMmbed/mbed-os/blob/master/CONTRIBUTING.md).
-- This is just a template, so feel free to use/remove the unnecessary things
+# Description
 
-## Description
+> Detailed changes summary | testing | dependencies
+> Good example: https://os.mbed.com/docs/latest/reference/guidelines.html#workflow
```

Comment:
Is this meant to point to an example bug/issue/PR?

Reply:
Yes, my comment above:

> Note: workflow url should be updated to point to a good PR example (@AnotherButler Can you help to get that url to a specific section? We will create new PR to handbook shortly, will be referenced in this PR).

Reply:
@AnotherButler the workflow PR template was merged, when it is published so I can edit the line 4 (update url to point to it) ? Or is it fine with link as it is ? This is the last blocking part for this PR

Reply:
I noticed `Pull request template` is already in the docs, but I do not see any anchor that I could add here. Is workflow enough (users need to scroll a bit to get the info we would like to)